### PR TITLE
Added local storage feature on JavaScript

### PR DESCRIPTION
**Pull request by demesameneshoa**
## In this PR
JavaScript Data Storage Feature is Employed
- When the user changes the content of any input field, the data is saved to the local storage.
- When the user loads the page, if there is any data in the local storage the input fields are pre-filled with this data.

### CODE REVIEW

**Comment by demesameneshoa** [comment]
I was arranged to work solo (please see the screenshot below)

image

File: script.js [hunk 1]
```diff
@@ -187,4 +187,51 @@ contactForm.addEventListener('submit', (event) => {
...
+  }));
+});
+
+messageInput.addEventListener('change', () => {
+  localStorage.setItem('contactData', JSON.stringify({
+    name: nameInput.value,
+    email: emailInput.value,
+    message: messageInput.value,
+  }));
+});
```
Comment (souad988) [inline script.js:221]: [OPTIONAL]Good job so far ,but I would suggest that you get data from local storage on page load put it in an object then on change you update only the item that has changed and use it to set localStorage item 

window.onload = () => {
  let FormData = JSON.parse(localStorage.getItem('contactFormInput')) || {name:"",email:"",message:""};
 nameInput.value = FormData.name;
  emailInput.value = FormData.email;
  messageInput.value = FormData.message;
};

**Review by souad988 — APPROVED** [review]
Hi @demesameneshoa , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Professional README
Good PR description
No linter errors

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by christianbiring1 — APPROVED** [review]
Hi @demesameneshoa , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS: APPROVED :tada: 
congraaaa

## Highlights
- [ ] Great work on preserving the data in the local Storage and retrieving them correctly on page reload  :1st_place_medal: 
- [ ] Descriptive PR `title` and `summary` :+1: 
- [ ] All linters checks are passing :+1: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**